tests/test-admin-locking.sh

```diff
@@ -25,10 +25,10 @@ set -euo pipefail
 setup_os_repository "archive-z2" "syslinux"
 
 # If parallel is not installed, skip the test
-if ! parallel --help >/dev/null 2>&1; then
-    echo "1..0 # SKIP no /usr/bin/parallel"
+if ! which parallel >/dev/null 2>&1; then
```

Comment:
OK, well it seems like the moreutils one doesn't even have --version, and exits 1 when given. So we can just replace that --help with --version. @smcv, should I make a commit for you to cherry-pick, or do you want to apply that directly on your PR?